apps/weshipit/pages/coding-retreat.tsx

```diff
@@ -0,0 +1,157 @@
+import { Button, Card, Hero, Prose, Text } from '@weshipit/ui';
+import { Layout } from '../components/layout';
+import React from 'react';
+
+export default function CodingRetreat() {
+  return (
+    <Layout
+      withFooter
+      seoTitle="Indie Hackers Coding Retreat"
+      seoDescription="Looking to take your project to the next level? Join our Indie Hackers Coding Retreat for a week of immersive coding, learning, and networking. Get personalized guidance from experienced developers and connect with like-minded indie hackers."
+      ogImageTitle="Join our next coding retreat!"
+      withAccessoryRight={
+        <Button
+          as="a"
+          variant="primary"
+          href="https://airtable.com/appLcVC7NmRBu1itw/shrXRYspfLCX6m1Q8"
```

Comment:
use `coding_retreat` database

![CleanShot 2024-05-16 at 12 17 18](https://github.com/flexbox/weshipit.today/assets/360936/0403110c-1d6e-41e5-a489-9b8066f65fb0)


and export link [from `api/links.ts`](https://github.com/flexbox/weshipit.today/blob/9daba70e2849cd2ba2b6cca49c3cea945bb6f39e/apps/weshipit/pages/api/links.ts#L1)